### PR TITLE
Add permissions to motion_statute_paragraph. Update tests.

### PR DESCRIPTION
**Pull request by reiterl**

### CODE REVIEW

File: tests/system/action/motion_statute_paragraph/test_sort.py [hunk 1]
```diff
@@ -67,3 +68,36 @@ def test_sort_another_section_db(self) -> None:
...
+            {
+                "motion_statute_paragraph/31": {
+                    "meeting_id": 1,
+                    "title": "title_loisueb",
+                },
+                "motion_statute_paragraph/32": {
+                    "meeting_id": 1,
+                    "title": "title_blanumop",
+                },
+            },
```
Comment (jsangmeister) [inline tests/system/action/motion_statute_paragraph/test_sort.py:83]: See https://github.com/OpenSlides/openslides-backend/pull/603#discussion_r602177045

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: tests/system/action/motion_statute_paragraph/test_sort.py (hunk 1 above)
Comment (reiterl) [reply tests/system/action/motion_statute_paragraph/test_sort.py]: Add permission_test_model in setup.

**Review by jsangmeister — CHANGES_REQUESTED** [review]
Please change permission strings to the enum

**Review by jsangmeister — APPROVED** [review]
Whoops, just saw https://github.com/OpenSlides/openslides-backend/pull/603#discussion_r603213092...

**Pull request merged**